src/Benchmarks/Middleware/PlaintextMiddleware.cs

```diff
@@ -37,11 +37,15 @@ public static Task WriteResponse(HttpResponse response)
             var payloadLength = _helloWorldPayload.Length;
             response.StatusCode = 200;
             response.ContentType = "text/plain";
-            response.ContentLength = payloadLength;
+
+            // HACK: Setting the Content-Length header manually avoids the cost of serializing the int to a string.
+            //       This is instead of: httpContext.Response.ContentLength = payloadLength;
+            response.Headers["Content-Length"] = "13";
```

Comment:
I think we want to keep the code in Benchmarks: https://github.com/aspnet/benchmarks/commit/19ecf5ee33e3cda64ec696b6c76cd9fea22d84e1